docs/_docs/admin-guide/admin-installation.md

```diff
@@ -48,7 +48,39 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
 Alternatively, you can edit the `CODENVY_HOST` value in `codenvy.env`. 
 
 ## Ports
-TODO - UPDATE PORT DESCRIPTION SECTION
+The master node is where Codenvy is installed and running. In a [scalability mode]({{base}}/docs/admin-guide/managing/index.html), you can add additional physical "workspace" nodes to increase system capacity. If you have not added any additional physical workspace nodes, then the Codenvy master node doubles as a workspace node and needs both sets of ports opened.
+
+#### Master Node: External
+|Port|Service|Notes|
+|---|---|---|
+|80 / 443|HAProxy HTTP/S|HTTP is the default. If you configure [HTTP/S]({{base}}/docs/admin-guide/configuration/index.html#https), then port 80 can be closed.
+|5000|Docker Registry|Embedded registry to save workspace snapshots. This port is not required if you configure an external registry.
```

Comment:
You do not need port 5000 on a workspace node. Just master node, and only if you have at least one workspace node. Moreover, this port may be available just for the nodes and closed for other connections.